microsoft-edge/devtools-guide-chromium/device-mode/index.md

```diff
@@ -176,6 +178,11 @@ If the mobile device option that you need isn't included on the default list, yo
 
    ![Adding a custom device](./index-images/device-mode-toggle-device-toolbar-settings-emulated-devices-add.png)
 
+1. Click **Add** to add the new custom device. The new device should now be available in the **Device** list.
```

Comment:
```suggestion
1. Click **Add** to add the new custom device. The new device is now available in the **Device** list.
```
simplified wording, optimistic (happy path; assume success) wording
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/should-vs-must

Reply:
applied locally in VS Code